asciinema2gif

```diff
@@ -118,3 +118,5 @@ convert -delay 5 -loop 0 frames/*.png gif:- | gifsicle --colors=256 --delay=6 --
 cd - >/dev/null
 mv "${tempdir}/asciicast.gif" "${output_file}"
 echo ">> Generated in: ${output_file}"
+echo "cleaning up the frames before quiting"
```

Comment:
To be consistent with the rest of the script, this should end in a period. It should also start with an uppercase letter. Finally, there are two “t” in “quitting”. `Cleaning up the frames before quitting.`

All that said, deleting the directory will be fairly fast and it’s not like the user will be left waiting for it. I don’t think this message adds value to the user, and might add confusion. I’d remove it and just perform the deletion.